### PR TITLE
fix(Components): List actions appears on hover on the right above the tr content

### PR DESCRIPTION
**Pull request by jsomsanith**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
Actions next to the item title

screen shot 2017-02-10 at 17 35 48

screen shot 2017-02-10 at 17 38 46

**What is the new behavior?**
Actions overlay on the right
screen shot 2017-02-10 at 17 35 16

screen shot 2017-02-10 at 17 38 34

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

**Other information**:
The actions are still next to the item title in the DOM for accessibility.

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://140.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://140.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://140.talend.surge.sh)

File: packages/components/src/List/DisplayTable/DisplayTable.scss [hunk 1]
```diff
@@ -93,9 +94,16 @@ $tc-list-table-header-height: 36px !default;
...
 					flex-shrink: 0;
 					flex-grow: 0;
-					width: 0;
+					opacity: 0;
+					visibility: hidden;
+
+					position: absolute;
+					right: 0;
+					background: linear-gradient(to right, transparent, $tc-list-table-hover-bg 20px);
+					padding-left: 20px;
```
Comment (jmfrancois) [inline packages/components/src/List/DisplayTable/DisplayTable.scss]: please use variable

**Comment by frassinier** [comment]
Background is needed for actions because of hovering will be the messreact storybook

File: packages/components/src/List/DisplayLarge/DisplayLarge.scss [hunk 2]
```diff
@@ -20,7 +24,13 @@ $tc-list-display-large-color: #AAA !default;
 	}
 
 	:global(.tc-actions) {
+		transition: opacity 0.15s ease-in;
+		opacity: 0;
 		visibility: hidden;
+		position: absolute;
+		right: 0;
+		background: linear-gradient(to right, transparent, $tc-list-display-large-background-color 20px);
+		padding-left: 20px;
```
Comment (jmfrancois) [inline packages/components/src/List/DisplayLarge/DisplayLarge.scss]: sorry but same here

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://140.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://140.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://140.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://140.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://140.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://140.talend.surge.sh)

**Pull request merged**